src/app/header/header.component.html

```diff
@@ -1,20 +1,30 @@
-<nav class="navbar navbar-default">
+<nav class="navbar navbar-default" role="navigation">
   <div class="container-fluid">
     <div class="navbar-header">
-      <a href="#" class="navbar-brand">Tuiter</a>
+      <a href="#" rel="home" class="navbar-brand">
+          <img src="../../assets/tuiter.png">
+      </a>
     </div>
 
     <div class="collapse navbar-collapse">
       <ul class="nav navbar-nav">
-        <li routerLinkActive="active"><a routerLink="/profile">Profile</a></li>
+        <li routerLinkActive="active">
+          <a routerLink="/profile">
+            &nbsp;<span class="glyphicon glyphicon-user"></span>&nbsp;&nbsp;Informação Pessoal
+          </a>
+        </li>
+          <li routerLinkActive="active">
+              <a routerLink="/update-pass">
+                <span class="glyphicon glyphicon-lock"></span>&nbsp;&nbsp;Segurança e Privacidade
+          </a>
+        </li>
       </ul>
       <ul class="nav navbar-nav navbar-right">
         <li class="dropdown" appDropdown>
-          <a style="cursor: pointer;" class="dropdown-toggle" role="button">Manage <span class="caret"></span></a>
+          <a style="cursor: pointer;" class="dropdown-toggle" role="button"><span class="glyphicon glyphicon-triangle-bottom"></span></a>
```

Comment:
I always prefer to write style in the scss file, but leave this comment only as a suggestion.